src/modules/jcms_migrate/jcms_migrate.install

```diff
@@ -1699,3 +1699,48 @@ function jcms_migrate_update_8160() {
   $migrate_id_map = \Drupal::service('plugin.manager.config_entity_migration')->createInstance('jcms_subjects_json')->getIdMap();
   $migrate_id_map->delete(['human-biology-medicine']);
 }
+
+/**
+ * Enable config_import module.
+ */
+function jcms_migrate_update_8161() {
+  drupal_flush_all_caches();
+  \Drupal::service('module_installer')->install(['config_import']);
+}
+
+/**
+ * Update config to introduce new text format basic_html_plus.
+ */
+function jcms_migrate_update_8162() {
+  $config_importer = \Drupal::service('config_import.importer');
+  $configs = [
+    'filter.format.basic_html_plus.yml',
```

Comment:
This text format needs to be available to switch the format of fields on existing job adverts.